gravitee-apim-cypress/cypress/integration/apim/rest-api/apis/swagger-import-tests.ts

```diff
@@ -0,0 +1,269 @@
+        .its('body')
+        .should('have.length', 7)
+        .its('0.name')
+        .should('equal', 'Cat 1');
+    });
+  });
+
+  describe('Tests mock path policy', () => {
+    after(() => cy.teardownApi(mockPolicyApi));
+
+    it('should get a mocked response when trying to reach API endpoint', () => {
+      requestGateway({ url: `${Cypress.env('gatewayServer')}${mockPolicyApi.context_path}/pet/findByStatus?status=available` })
+        .its('body.category.name')
+        .should('equal', 'Mocked string');
+    });
+  });
+
+  describe('Tests JSON-Validation path policy', () => {
+    after(() => cy.teardownApi(jsonValidationPolicyApi));
+
+    it('should fail with BAD REQUEST (400) when sending data using an invalid JSON schema', () => {
+      requestGateway(
+        {
+          url: `${Cypress.env('gatewayServer')}${jsonValidationPolicyApi.context_path}/pet`,
+          method: 'PUT',
+          body: {
+            invalidProperty: 'invalid value',
+          },
+        },
+        {
+          validWhen: (response) => response.status === 400,
+        },
+      ).should((response) => {
+        expect(response.body.message).to.equal('Bad Request');
+      });
+    });
+
+    it('should successfully connect to API endpoint if JSON schema is valid', () => {
+      const body = {
+        id: 2,
+        category: {
+          id: 0,
+          name: 'string',
+        },
+        name: 'doggie',
+        photoUrls: ['string'],
+        tags: [
+          {
+            id: 0,
+            name: 'string',
+          },
+        ],
+        status: 'available',
+      };
+      requestGateway(
+        {
+          url: `${Cypress.env('gatewayServer')}${jsonValidationPolicyApi.context_path}/pet`,
+          method: 'PUT',
+          body,
+        },
+        {
+          validWhen: (response) => response.status === 200,
+        },
+      ).should((response) => {
+        expect(response.body.name).to.equal('doggie');
+      });
+    });
+  });
+
+  describe('Tests XML-Validation path policy', () => {
+    after(() => cy.teardownApi(xmlValidationPolicyApi));
+
+    it('should fail with BAD REQUEST (400) when sending data using an invalid XML schema', () => {
+      requestGateway(
+        {
+          url: `${Cypress.env('gatewayServer')}${xmlValidationPolicyApi.context_path}/pet`,
+          method: 'PUT',
+          headers: {
+            'Content-Type': 'application/xml',
+          },
+          body: {
+            invalidProperty: 'invalid value',
+          },
+        },
+        {
+          validWhen: (response) => response.status === 400,
+        },
+      ).should((response) => {
+        expect(response.body.message).to.equal('Bad Request');
+      });
+    });
+
+    // test not working yet, needs investigation to figure out if there's an issue with the gateway
+    it.skip('should successfully connect to API endpoint if XML schema is valid', () => {
```

Comment:
I saw that this code has been moved. I wonder if the issue is still there? 🤔

Reply:
It's still there, for now